draft-ietf-jsonpath-base.md

```diff
@@ -1043,29 +1043,32 @@ logical-or-expr  = logical-and-expr *("||" logical-and-expr)
 logical-and-expr = basic-expr *("&&" basic-expr)      ; conjunction
                                                       ; binds more tightly than disjunction
 
-basic-expr   = comp-expr / existence-expr / paren-expr / (neg-op paren-expr)
-existence-expr = [neg-op] path                        ; path existence or non-existence
+basic-expr   = exist-expr / paren-expr / (neg-op paren-expr) / relation-expr
+exist-expr   = [neg-op] path                          ; path existence or non-existence
+path         = rel-path / json-path
+rel-path     = "@" *(dot-selector / index-selector)
```

Comment:
Are we only supporting a single selector here, or does the `*( ... )` mean multiple?

Reply:
`*()` means zero or more repetitions of the contents of the brackets. See [section 3.6](https://www.rfc-editor.org/rfc/rfc5234.html#section-3.6) of RFC 5234.